client/mirage/factories/project.js

```diff
@@ -1,33 +1,130 @@
 import { Factory, trait } from 'ember-cli-mirage';
 
+const PACKAGE_STATUS_CODES = {
+  PACKAGE_PREPARATION: 'Package Preparation',
+  CERTIFIED: 'Certified',
+  FINAL_APPROVAL: 'Final Approval',
+  REVIEWED_NO_REVISIONS_REQUIRED: 'Reviewed - No Revisions Required',
+  REVIEWED_REVISIONS_REQUIRED: 'Reviewed - Revisions Required',
+  WITHDRAWN: 'Withdrawn',
+  SUBMITTED: 'Submitted',
+  UNDER_REVIEW: 'Under Review',
+  MISTAKE: 'Mistake',
+};
```

Comment:
Not sure if I"m missing something, but you can export these from the original controller class file like so:

`export const PACKAGE_STATUS_CODES = {`

Then you'd be able to import them here:

`import { PACKAGE_STATUS_CODES } from '../find/correct/path/whatever-file-name-is';`